microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -303,6 +303,66 @@ The first number in the group of numbers for an edge in the `edges` array corres
 | Shortcut | An easier-to-read representation of some other path. This type is rarely used. For example, if you use `Function.prototype.bind` to create a bound function with some bound arguments, V8 creates a `JSBoundFunction`, which points to a `FixedArray` (an internal type), which points to each bound argument. When producing a snapshot, V8 adds a shortcut edge from the bound function directly to each bound argument, bypassing the `FixedArray`. |
 | Element | Object properties where the key is a number. |
 
+#### locations
+
+The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of `HeapSnapshot` nodes.
+
+To use this array, the following information is needed:
+
+* `snapshot.meta.location_fields`, to know how many fields each location group has.
+* `snapshot.nodes`, to link a location group back to a node.
+
+The current `snapshot.meta.location_fields` has the information for each location group:
+| Index in location_fields | Name | Description |
```

Comment:
```suggestion
| Index in `location_fields` | Name | Description |
```
assuming this produces clear consistent result in rendered article

gh rendered section:
https://github.com/christyycchen/edge-developer/blob/patch-1/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md#locations